tests/test_polygon_class.py

```diff
@@ -0,0 +1,27 @@
+import h3
+
+
+def test_repr():
+    a = '8928308280fffff'
+    b = h3.grid_ring(a, 5).pop()
+
+    cells1 = h3.grid_ring(b, 2) | {a}
+    cells2 = cells1 | {b}
+
+    poly1 = h3.cells_to_polygons(cells1)
+    poly2 = h3.cells_to_polygons(cells2)
+
+    # unfortunately output order is nondeterministic
```

Comment:
Why is it that the output order of the polygons is nondeterministic? @isaacbrodsky @dfellis @nrabinowitz

Reply:
Do you actually mean nondeterministic, or just undefined? There's nothing in the C lib that should be nondeterministic here, there's just no guarantee of order

Reply:
Linking what you answered here: https://github.com/uber/h3-py/issues/277#issuecomment-1222662264

Reply:
It's actually non-deterministic. I'll get different output orders across different runs for the same input.

Reply:
Running the same code (after restarting the interpreter):

```python
import h3

a = '8928308280fffff'
b = h3.grid_ring(a, 5).pop()

cells1 = h3.grid_ring(b, 2) | {a}
cells2 = cells1 | {b}

poly1 = h3.cells_to_polygons(cells1)
poly2 = h3.cells_to_polygons(cells2)

print(poly1)
print(poly2)
```

I got these different outputs:

```python
[<h3.Polygon |outer|=6, |holes|=()>, <h3.Polygon |outer|=30, |holes|=(18,)>]
[<h3.Polygon |outer|=6, |holes|=()>, <h3.Polygon |outer|=30, |holes|=(18,)>, <h3.Polygon |outer|=6, |holes|=()>]

[<h3.Polygon |outer|=30, |holes|=(18,)>, <h3.Polygon |outer|=6, |holes|=()>]
[<h3.Polygon |outer|=30, |holes|=(18,)>, <h3.Polygon |outer|=6, |holes|=()>, <h3.Polygon |outer|=6, |holes|=()>]

[<h3.Polygon |outer|=30, |holes|=(18,)>, <h3.Polygon |outer|=6, |holes|=()>]
[<h3.Polygon |outer|=6, |holes|=()>, <h3.Polygon |outer|=30, |holes|=(18,)>, <h3.Polygon |outer|=6, |holes|=()>]
```

Reply:
Ahhh. I think it is because Python is not deterministic with set ordering. The following code seems to give consistent results:

```python
import h3

a = '8928308280fffff'
b = '89283082bc7ffff'
assert h3.grid_distance(a, b) == 5

cells1 = h3.grid_ring(b, 2) | {a}
cells2 = cells1 | {b}

cells1 = sorted(cells1)
cells2 = sorted(cells2)

poly1 = h3.cells_to_polygons(cells1)
poly2 = h3.cells_to_polygons(cells2)

print(poly1)
print(poly2)
```